.prettierrc

```
@@ -6,5 +6,5 @@
   "trailingComma": "all",
   "arrowParens": "avoid",
   "endOfLine": "auto",
-  "proseWrap": "always"
+  "proseWrap": "preserve"
```

Prose wrap should be staying as `always` to enforce the line max print with. These will result in easier to read PRs on GitHub.

```suggestion
  "proseWrap": "always"
```

---

Not sure if you missed the comment about `proseWrap` above. Happy to change it but as mentioned this PR will be *massive* otherwise

---

Yes I saw your comment about it. I'm not afraid about large diffs! Especially when they are for formatting :)